app/app.h

```diff
@@ -9,6 +9,39 @@ namespace vsite::oop::v2
 /*
 * class declaration
 */
+	class color
+	{
+	private:
+		double red;
+		double green;
+		double blue;
+	public:
+		color() {	// constructor za null vrijednos
```

Comment:
Čemu još jedan konstruktor, zašto niste samo postavili defaultne vrijednosti za r/g/b u konstruktor sa vrijednostima?